packages/webdriverio/src/commands/element/click.ts

```diff
@@ -84,79 +84,93 @@ const log = logger('webdriver')
  * @param {number=}           options.y      Number (optional)
  * @param {boolean=}           options.skipRelease         Boolean (optional)
  */
-export async function click(
+export function click(
     this: WebdriverIO.Element,
-    options?: ClickOptions
+    options?: Partial<ClickOptions>
 ) {
-    if (typeof options === 'undefined') {
-        return this.elementClick(this.elementId)
+    if (typeof options !== 'undefined') {
+        if (typeof options !== 'object' || Array.isArray(options)) {
+            throw new TypeError('Options must be an object')
+        }
+        return actionClick(this, options)
+    }
+
+    return elementClick(this)
+}
+
+/**
+* Workaround function, because sometimes browser.action().move() flaky and isn't able to scroll pointer to into view
+* Moreover the action  with 'nearest' behavior by default where element is aligned at the bottom of its ancestor.
+* and could be overlapped. Scroll to center should definitely work even if element was covered with sticky header/footer
+*/
+async function workaround(element: WebdriverIO.Element) {
+    await element.scrollIntoView({ block: 'center', inline: 'center' })
+}
+
+async function elementClick(element: WebdriverIO.Element) {
+    try {
+        return element.elementClick(element.elementId)
```

Comment:
same here
```suggestion
        await element.elementClick(element.elementId)
```

Reply:
We previously also returned the click, we expect it in our tests as sometimes the result is expected to be `null`

Reply:
You're absolutely right, thanks for catching this. Never knew that this could not be handled like this.

Pushed a fix in which I use `return await`